### ISSUE TITLE
Error in SPAGMMATtest, samples used in glmm model fit do not have dosages

### ISSUE THREAD

**Issue by ConnieXuhm** [weizhouUMICH/SAIGE#268]
Hi,I met a problem when conducting association analysis step2. 
Here is my code
```
SPAGMMATtest(vcfFile="mymerged_maf.vcf.gz", vcfFileIndex="mymerged_maf.vcf.gz.tbi", chrom="1", vcfField="GT",minMAF=0.0001, minMAC=1, sampleFile="IDdosage", GMMATmodelFile="1103.rda", varianceRatioFile="1103.varianceRatio.txt", SAIGEOutputFile="Output.vcf.genotype.txt", numLinesOutput=2, IsOutputAFinCaseCtrl=TRUE )
```
Then an error got like this
```
[1] "Leave-one-chromosome-out is not applied"
Single variance ratio is provided, so categorical variance ratio won't be used!
variance Ratio is  1 
isCondition is  FALSE 
Open VCF done
To read the field GT
Number of meta lines in the vcf file (lines starting with ##): 28
Number of samples in in the vcf file: 2815
2815  sample IDs are found in the vcf file
Error in SPAGMMATtest(vcfFile = "mymerged_maf.vcf.gz", vcfFileIndex = "mymerged_maf.vcf.gz.tbi",  : 
  ERROR!2815 samples used in glmm model fit do not have dosages
```

The input file is coverted from bim bed fam format by plink
I want to know how to fix the no-dosage problem, and whether the step1 variance ratio= 1 is normal.
Thanks a lot!

**Comment by weizhou0** [comment]
Hi @ConnieXuhm, 

It seems the sample IDs in the VCF file do not match the sample IDs in the plink file in Step 1. Could you please have a check? 

Thanks,
Wei

**Comment by ConnieXuhm** [comment]
> Hi @ConnieXuhm,
> 
> It seems the sample IDs in the VCF file do not match the sample IDs in the plink file in Step 1. Could you please have a check?
> 
> Thanks,
> Wei

Dear  @weizhou0 , thanks for your reply!
The VCF file I used was just converted from the step 1 plink files by 
``` ./plink --bfile 1202_step1 --export vcf --out 1202_step2 ```
however, there is still the dosage problem.
I agree with that the problem may occur during the format conversion. Do you have any suggestion about how to convert the IMPUTE2 (.gen) format to .vcf or plink format?
Many thanks for your reply!

**Comment by privefl** [comment]
I had the same issue.
For some reason, the VCF file converted the sample IDs to `<FID>_<IID>`.
Then, to solve this issue, you can `load()` the `.rda` file from Step1, change `$sampleID` accordingly, and `save()` it again.

**Comment by weizhouUMICH** [comment]
Sorry fo the late reply! We have just released a new version 1.0.0. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
